airbyte-integrations/connectors/source-ip2whois/source_ip2whois/spec.yaml

```diff
@@ -0,0 +1,25 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/ip2whois
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Ip2whois Spec
+  type: object
+  required:
+    - access_key
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API key
+      type: string
+      description: >-
+        Your API Key. See <a
+        href="https://www.ip2whois.com/developers-api">here</a>.
+      airbyte_secret: true
+    domain:
+      title: Domain
+      type: string
+      description: >-
+        Domain name. See <a
+        href="https://www.ip2whois.com/developers-api">here</a>.
+      examples:
+        - www.google.com
+        - www.facebook.com
```

Comment:
Why not use this as an array and use the ListStream feature to read records from multiple domains?